.github/dependabot.yml

```diff
@@ -18,3 +18,8 @@ updates:
           # We are pinned to Tachometer 0.5.10 due to a breaking change in 0.6.0.
           # See: https://github.com/google/tachometer/issues/244
           - dependency-name: 'tachometer'
+          # TODO [#]:
```

Comment:
Let's add an actual issue here

Reply:
![i did it](https://github.com/user-attachments/assets/58a6a470-a49c-4911-9a18-61a6ba56b00d)
heheheh